.changeset/fresh-pots-draw.md

```diff
@@ -0,0 +1,5 @@
+---
+'astro': patch
```

Comment:
Can you make this be `minor`?

Reply:
changed to minor and copied your very good example from the docs. Feel free to improve the text as I am not a native speaker (well at least not of English)